clusterloader/e2e.go

```diff
@@ -0,0 +1,202 @@
+/*
```

Comment:
Please add a comment just below `package` which says that this is basically a fork.